admin_manual/maintenance/manually-moving-data-folders.rst

```diff
@@ -41,15 +41,25 @@ run the SQL below:
 .. code-block:: sql
    
   UPDATE oc_storages SET id='local::/mnt/owncloud' 
-    WHERE id='local::/var/www/owncloud/data'
+    WHERE id='local::/var/www/owncloud/data';
 
-The other area to check is the `oc_jobs` table. The logrotate process may have
-hard-coded a non-standard (or old) value for the data path. To check it, run the
-SQL below and see if any results are returned:
+You next need to update the ``home`` column in the ``oc_accounts`` table. 
+This column contains the absolute path for user folders, e.g., ``/mnt/data/files/admin``.
+Assuming that the new home directory is: ``/mnt/data/files/super-admin``, and that the user’s id is 1, you could change it using the following SQL statement:
+
+.. code-block:: sql
+   
+  UPDATE oc_accounts SET home='/mnt/data/files/super-admin' 
```

Comment:
Actually, after reviewing the existing content, it makes sense to either keep the existing proposed changes and add a note, or to remove all the other SQL examples; to help avoid the same issue you've highlighted. I'd honestly prefer to add a note wording it to take into account what you've suggested.